src/matrix/layout_info.cpp

```diff
@@ -60,8 +60,8 @@ SizeType LayoutInfo::minMemSize() const noexcept {
 }
 
 SizeType LayoutInfo::minTileMemSize(const TileElementSize& tile_size) const noexcept {
-  DLAF_ASSERT_HEAVY(tile_size.rows() <= block_size_.rows(), "");
-  DLAF_ASSERT_HEAVY(tile_size.cols() <= block_size_.cols(), "");
+  DLAF_ASSERT_HEAVY(tile_size.rows() <= block_size_.rows(), tile_size.rows(), block_size_.rows());
+  DLAF_ASSERT_HEAVY(tile_size.cols() <= block_size_.cols(), tile_size.cols(), block_size_.cols());
```

Comment:
Just place `tile_size` and `block_size` here, the more info the better. We discussed this in #295 .